portfolio/src/main/webapp/script.js

```diff
@@ -28,12 +28,26 @@ function addRandomGreeting() {
 }
 
 /**
- * Fetches random quote from the server
- * Currently just gets a hardcoded message from the server
+ * Fetches random quote from the server.
+ * Currently just gets a hardcoded message from the server.
  * TODO(psharma): Update this method to use Lambdas.
  */
 function getRandomQuote() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('quote-container').innerText = quote;
+    console.log("inside getrandomquote");
+  fetch('/data').then(response => response.json()).then((quote) => {
```

Comment:
Remove logging if you don't need it. I am ok with it if you want to keep it around for the next step in the tutorial.
If you are planning to keep it, make sure the indent is correct. I would suggest running the linter.